.drone.yml

```diff
@@ -68,21 +68,7 @@ pipeline:
       # @todo: Re-run line below after port to ECL 2.0 is completed.
       # - ./vendor/bin/phpunit
       # @todo: Manually add tests to run until port to ECL 2.0 is completed.
```

Comment:
I think this comment is outdated now, since we use the `ecl1` tag. It should probably read:
```
       # @todo: remove the `@group ecl1` on PHPUnit tests to have them executed. We will remove this after porting is completed.
```

Reply:
good catch